### DISCUSSION TITLE
Help needed: Crashes when action !timeS is used. How to troubleshoot?

### DISCUSSION THREAD

**Discussion by cprima** [gopro/labs#307] [Q&A] [answered]
Problem:
I want to have a barcode that starts a video after 20seconds of delay, for 60 seconds duration.
1. the GoPro Hero11 with VERS	8, 2, 2    FMWR	H22.01.02.10.70 crashes / reboots
2. and I dont understand how to troubleshoot (I assume there are conflicting settings that trigger the crash)

Steps to reproduce:
1. Factory reset via on screen menu
2. !RESET (not sure if needed)
3. set a baseline of options https://gopro.github.io/labs/control/set/?cmd=!MPRXY=4!MHIST=1!MBITR=190!MLOGB=400!MNR01=1!MEXPX=180!MEXPS=1mPRoB4C5D4E1FFG1L0Q0R1S1V4VC1W0X0 
4. set owner information https://gopro.github.io/labs/control/set/?cmd=%21MOWNR%3D%22Christian_Prior-Mamulyan%5Cneheleute%40archam.de%5Cn%2B49%20176%204795%203335%22
5. setting some Camera-specific stuff https://gopro.github.io/labs/control/set/?cmd=oR2!MBASE="cpmCAM03_[yyyy-mm-dd]_"
6. overwriting (and activating) the Standard Video mode setting https://gopro.github.io/labs/control/set/?cmd=mV0r5p30fHte1hS0dR0aAcFd1b1wAi8M1x-.5sLa
7. **Triggering a start in 20 seconds** https://gopro.github.io/labs/control/set/?cmd=%21MDSPL%3D1mV%2120N%21S%2160E

Workaround:
1. Factory reset via on screen menu
2. **Triggering a start in 20 seconds** https://gopro.github.io/labs/control/set/?cmd=%21MDSPL%3D1mV%2120N%21S%2160E

So my assumption is that I should refrain from setting one of the (many) other settings. **But which?**
Can I write out "traces"/stackdumps/…?

**Comment by dnewman-gpsw** [comment, marked as answer]
Sorry, this has taken me a while.  

The clash is between scripted starts (commands like !20S) and !MEXPS=1 and/or !MHIST=1.  I don't know why yet.  Weird triggered starts in boot commands are working fine with EXPS and HIST active.

**Comment by cprima** [comment]
Oh wow, that is a great ★★★★★ answer, thank you! I started using GoPro cameras only recently and the access to (de facto) "firmware support" here on GitHub is outstanding. Tell your product manager they have a great thing going on in here, and that I bought not one but two Hero 11 because of "Labs" and 10bit: This is "CHDK for moving pictures". 😍

I have some well-trained IT troubleshooting skills, but would not have suspected a supposedly "display-only" setting.
Removed the two extensions from my baseline QR codes and will go on a weekend bicycle trip to capture some test footage.

**Comment by cprima** [comment]
![IMG_20230425_141935](https://user-images.githubusercontent.com/943871/234276734-58efa550-a2d3-482c-b176-fa14ab46681e.jpg)

Here is how I used the QR-Codes on the bicycle tour last weekend.
(I find it easier to wave a laminated paper in front of the camera than using the app,
and I can store the URLs to the QR-Code-Generator in a spreadsheet.)
